.github/workflows/slovensko_digital_ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Slovensko.Digital CI
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    if: ${{github.ref == 'refs/heads/master'}}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        upvs_env: [dev, fix, prod]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: dokku/github-action@master
+        with:
+          git_remote_url: ssh://dokku@${{secrets.STAGING_HOST}}/${{matrix.upvs_env}}.${{github.event.repository.name}}
```

Comment:
Tu by som asi napevno hodil proste `ssh://dokku@slovensko.digital:${{github.event.repository.name}}` cc @pavolzbell ?

Reply:
@luciajanikova `STAGING_*` premmenne treba nahradit `SSD_PRODUCTION_HOST` a `SSD_PRODUCTION_KEY` ako som pisal v ECO-112 zaroven treba skontrolovat ci sa appka na dokku naozaj vola tak ako repo lebo nie vzdy je to zial tak (myslim cast `${{github.event.repository.name}}`)